src/Microsoft.AspNetCore.Mvc.Core/ControllerBase.cs

```diff
@@ -911,6 +911,120 @@ public virtual CreatedAtRouteResult CreatedAtRoute(string routeName, object rout
         }
 
         /// <summary>
+        /// Creates a <see cref="AcceptedResult"/> object that produces an Accepted (202) response.
+        /// </summary>
+        /// <param name="uri">The URI with the location at which the status of requested content can be monitored.</param>
+        /// <param name="value">The optional content value to format in the entity body.</param>
+        /// <returns>The created <see cref="AcceptedResult"/> for the response.</returns>
+        [NonAction]
+        public virtual AcceptedResult Accepted(string uri, object value = null)
```

Comment:
We don't use optional values in any of our public API. http://haacked.com/archive/2010/08/10/versioning-issues-with-optional-arguments.aspx has some background for why it's a bad idea. You'll have to add additional overloads that don't take `value`